Formula/hasura-cli.rb

```diff
@@ -6,6 +6,7 @@ class HasuraCli < Formula
   url "https://github.com/hasura/graphql-engine/archive/v2.0.10.tar.gz"
   sha256 "ca134148d62985a065705740a7ca884a2f1bfe18c1f11d21b66dc17119630ec5"
   license "Apache-2.0"
+  revision 1
```

Comment:
Is the current bottle broken? If not, we can omit the revision bump.

Reply:
yea realized that after. assuming not since `brew test hasura-cli` is happy. fixed